### DISCUSSION TITLE
【灌水】这个委托我接了

### DISCUSSION THREAD

**Discussion by fuqiam** [share121/inter-knot#1693] [其他]
有自动接委托的脚本吗，最近委托根本抢不到，快要饿死了

**Comment by lllliufeng** [comment]
[自动接委托.exe](https://www.bilibili.com/video/BV1GJ411x7h7/)

**Reply by yutong903** [reply]
被骗的最惨的一次=（

**Comment by Inter-Knot-User** [comment]
> [自动接委托.exe](https://www.bilibili.com/video/BV1GJ411x7h7/)

好的自动接委托，使我的绳网飞速旋转
